doc/integrator/tilegeneration.rst

```diff
@@ -122,3 +122,9 @@ If you setup all the default options, you can generate the tiles by using the co
 .. prompt:: bash
 
     docker-compose exec tilecloudchain generate_tiles
+
+AWS credentials
+~~~~~~~~~~~~~~~
+
+To be able to connect to the S3 service sou should define the folowing variable in the ``.env.mako`` file:
```

Comment:
service sou -> service, you
folowing variable -> following variables

Reply:
Thanks :-)